recipes/gridmarthe/meta.yaml

```
@@ -0,0 +1,69 @@
+{% set name = "gridmarthe" %}
+{% set version = "0.1.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://gitlab.com/brgm/hydrogeological-modelling/marthe-tools/gridmarthe/-/archive/v{{ version }}/gridmarthe-v{{ version }}.tar.gz
+  sha256: 19b9866f516bc3a45fc97d7cee408723fb6ce84bf01340938d86b66d1d1b5400
+
+build:
+  number: 0
+  skip: True  # [py<310]
+  # entry_points:
+  #     - ncmart = gridmarthe:scripts.ncmart
+  #     - cleanmgrid =gridmarthe:scripts.cleanmgrid
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('fortran') }}
+    - {{ stdlib("c") }}
+    - ninja
+    - meson
+  host:
+    - python
+    - pip
+    - charset-normalizer
+    - meson
+    - meson-python
+    - numpy >=1.26
```

If there's any code being compiled against numpy, you need to unpin this
```suggestion
    - numpy
```
(more details [here](https://conda-forge.org/docs/maintainer/knowledge_base/#building-against-numpy) - though I just noticed it's quite outdated, sorry)

---

OK, thanks ! Done in https://github.com/conda-forge/staged-recipes/pull/28277/commits/2515106d3cad30fe31004c7e571371b70eb4bde1
So, compared with the documentation, I do not need to the pinned numpy (` {{ pin_compatible('numpy') }}`) in run section anymore ?
It seems that with these modifications, the python 3.10 build find numpy 1.22 (in host section) and then failed for the 3 OS...